packages/shared/index.ts

```diff
@@ -0,0 +1,78 @@
+import { Signature } from "@chainsafe/aleo-snap-wasm";
+
+export { Signature } from "@chainsafe/aleo-snap-wasm";
```

Comment:
seems this import is duplicated

Reply:
exporting Signature in shared, so that example doesn't have to have aleo-snap-wasm dependency